tools/clang/test/SemaHLSL/hlsl/objects/HitObject/hitobject-in-buffer.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -T lib_6_9 %s -verify
+
+// expected-error@+1{{'HitObject' is an object and cannot be used as a type parameter}}
+RWStructuredBuffer<HitObject> InvalidBuffer;
+
+[shader("raygeneration")] void main() {
```

Comment:
Can this function be removed?

Reply:
Removed in [57c861a](https://github.com/microsoft/DirectXShaderCompiler/pull/7097/commits/57c861aab39debbbe5ba53dbb2a9bb4a7c40a822)